### PR TITLE
PI-2450: Replace calls to community API with calls to cvl and delius

### PR DESCRIPTION
**Pull request by pmcphee77**
This PR is an effort to move away from the deprecated Community API and replace calls to your dedicated cvl-and-delius API. Need to work out how we test this in the test environments before we release to prod.
**detekt** seems to be complaining due to the number of constructor params (which i havent changed, just renamed one)

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/CaCaseloadService.kt [hunk 1]
```diff
@@ -193,15 +194,15 @@ class CaCaseloadService(
...
 
     val caCaseListWithStaffUsername = cases.withStaffUsername.map { caCase ->
       val com = coms[caCase.probationPractitioner?.staffUsername?.lowercase()]
       if (com != null) {
         caCase.copy(
           probationPractitioner = ProbationPractitioner(
-            staffCode = com.staffCode,
-            name = com.staff?.let { "${it.forenames} ${it.surname}" },
+            staffCode = com.code,
+            name = com.name?.let { "${it.forename} ${it.surname}" },
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/CaCaseloadService.kt:205]: is there a reason this doesn't use `fullName`?

Comment (pmcphee77) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/CaCaseloadService.kt]: Great spot! Ive changed this. Also fixed detekt

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by andrewrlee — DISMISSED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**